bsp_diff/common/vendor/intel/external/project-celadon/dldt/0002-Enable-cldnn-and-cldnn_plugin-compiling-in-Celadon.patch

```diff
@@ -0,0 +1,48 @@
+From c34a367bd0db1bfac587a4b1444fea2a1de94222 Mon Sep 17 00:00:00 2001
+From: dengliqiang <liqiangx.deng@intel.com>
+Date: Mon, 23 Sep 2019 22:40:48 +0800
+Subject: [PATCH] Enable cldnn and cldnn_plugin compiling in Celadon
+
+Tracked-On: OAM-86770
+Signed-off-by: dengliqiang <liqiangx.deng@intel.com>
+---
+ inference-engine/src/cldnn_engine/cldnn_engine.cpp        | 5 +++++
+ inference-engine/src/cldnn_engine/cldnn_infer_request.cpp | 6 +++++-
+ 2 files changed, 10 insertions(+), 1 deletion(-)
+
+diff --git a/inference-engine/src/cldnn_engine/cldnn_engine.cpp b/inference-engine/src/cldnn_engine/cldnn_engine.cpp
+index 4b79fe6e..0f3fdf27 100644
+--- a/inference-engine/src/cldnn_engine/cldnn_engine.cpp
++++ b/inference-engine/src/cldnn_engine/cldnn_engine.cpp
+@@ -38,6 +38,11 @@ using InferenceEngine::Blob;
+ using namespace InferenceEngine;
+ using namespace details;
+ 
++#if defined(__ANDROID__)
++#define CI_BUILD_NUMBER "custom-master-android-nn"
```

Comment:
May be cl specific build number would be good

Reply:
For inference engine and MKLDNN, we also see:
dldt/inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp:11:#define CI_BUILD_NUMBER "custom-master-android-nn"
dldt/inference-engine/src/inference_engine/ie_version.cpp:9:#define CI_BUILD_NUMBER "custom-master-android-nn"
Then we have no idea about the detailed number